src/components/UpupUploader/FileBrowser/ListItem.tsx

```diff
@@ -51,33 +51,53 @@ const ListItem = ({
         ...(isFileSelected && { backgroundColor: backgroundColors.selected }),
         ...(!isFileSelected && { backgroundColor: backgroundColors.default }),
     }
-    const initial: Target = {
+
+    const transition: AnimationProps['transition'] = {
+        type: 'spring',
+        damping: 10,
+        stiffness: 100,
+        opacity: { delay: index * 0.05 },
+        y: { delay: index * 0.05 },
+    }
+
+    const initial: AnimationProps['initial'] = {
         opacity: 0,
         y: 10,
         ...backgroundColor,
     }
 
-    const animate: Target = {
+    const animate: AnimationProps['animate'] = {
         opacity: 1,
         y: 0,
         ...backgroundColor,
+        transition,
     }
```

Comment:
those one too ?

Reply:
I tried moving them outside the component but they require some vars from the component.

either way I don't like that code so we'll have to find a fix for the build error eventually